decidim-elections/app/forms/decidim/votings/admin/ballot_style_form.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module Admin
+      # A form to create/edit a ballot style
+      class BallotStyleForm < Form
+        attribute :title, String
```

Comment:
⚠️ This should be a translatable attribute, as the idea is to be able to show it in the frontend if needed. Also, I'm not 100% sure if we are going to use it, so we can remove it until is needed.

Reply:
🗑️   https://github.com/decidim/decidim/pull/7779/commits/193b53a55e0b776f6568020cfad28f02b22eaf5c